### PR TITLE
Wazo 2314 user moh on incall

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: wazo_confd/helpers/destination.py [hunk 1]
```diff
@@ -417,6 +417,34 @@ def make_user_fields_flat(self, data):
...
+
+        return data
+
+    @post_load
+    def merge_action(self, data):
+        ring_time = data.pop('ring_time', None)
+        moh_uuid = data.pop('moh_uuid', None)
+
+        actionarg2 = ''
+        if ring_time:
```
Comment (bloom1) [inline wazo_confd/helpers/destination.py:440]: What if `ring_time` is 0? Is that a use case?

Comment (pc-m) [reply wazo_confd/helpers/destination.py]: You are right, fixed!

File: wazo_confd/plugins/schedule/validator.py [hunk 2]
```diff
@@ -1,8 +1,107 @@
...
+
+        return data
+
+    @post_load
+    def merge_action(self, data):
+        ring_time = data.pop('ring_time', None)
+        moh_uuid = data.pop('moh_uuid', None)
+
+        fallback_actionargs = ''
+        if ring_time:
```
Comment (bloom1) [inline wazo_confd/plugins/schedule/validator.py:55]: Same thing here :smile:

Comment (pc-m) [reply wazo_confd/plugins/schedule/validator.py]: :see_no_evil:

**Review by bloom1 — APPROVED** [review]

**Pull request merged**